msgphp/user-bundle/0.2/src/Entity/User/User.php

```diff
@@ -2,14 +2,16 @@
 
 namespace App\Entity\User;
 
+use Doctrine\ORM\Mapping as ORM;
 use MsgPhp\User\Entity\User as BaseUser;
 use MsgPhp\User\UserIdInterface;
 
 /**
- * @final
+ * @ORM\Entity()
  */
 class User extends BaseUser
 {
+    /** @ORM\Id() @ORM\Column(type="msgphp_user_id") */
```

Comment:
I just saw this now. 

I’ve never seen two ORM at the same line. Does this work?

Reply:
Yep :+1: